packages/api/cms-api/src/user-permissions/entities/log-user.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, Property } from "@mikro-orm/core";
+import { EntityManager } from "@mikro-orm/postgresql";
+import { Field, ObjectType } from "@nestjs/graphql";
+
+import { LogUserPermission } from "./log-user-permission.entity";
+
+@ObjectType()
+@Entity({
+    expression: (em: EntityManager) => {
+        return em
+            .createQueryBuilder(LogUserPermission)
+            .select(["userId", "name", "email", "sum(usages) as usages", 'max("lastUsedAt") as lastseen'])
+            .groupBy(["userId", "name", "email"]);
```

Comment:
why group by userId, name and email?

I understand that you create new log entries to have kind of a backlog for changes to the name or email, but don't you want in this view all rows for a user?